system/default_ad_api/script/server.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 TIER IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+from threading import Thread
+
+from autoware_ad_api_msgs.srv import InterfaceVersion
+import flask
+import rclpy
+from rclpy.node import Node
+
+app = flask.Flask(__name__)
+cli = None
+
+
+@app.route("/interface/version")
+def interface_version():
+    req = InterfaceVersion.Request()
+    res = cli.call(req)
+    return flask.jsonify(convert_dict(res))
+
+
+def create_service(node, service_type, service_name):
+    cli = node.create_client(service_type, service_name)
+    while not cli.wait_for_service(timeout_sec=1.0):
+        node.get_logger().info(f"service not available, waiting again... ({service_name}")
+    return cli
+
+
+def convert_dict(msg):
+    fields = getattr(msg, "get_fields_and_field_types", None)
+    if not fields:
+        return msg
+    return {field: convert_dict(getattr(msg, field)) for field in fields()}
+
+
+def spin_ros_node():
+    global cli
+    rclpy.init(args=sys.argv)
+    node = Node("default_ad_api_server")
```

Comment:
I guess this is not a server for the default `ad_api` but the default web server of `ad_api`.
So I feel it's good to change the order of words and to explicit `web`.

```suggestion
    node = Node("ad_api_default_web_server")
```

Reply:
OK. I'll add web, but I think default_ad_api_web_server is suitable because default means default implementation of AD API.

Reply:
Hmm? But this server doesn't implement anything but just wraps them, right?
So I guess it can be used for non-default APIs as well. Am I wrong?

Reply:
Ah, I understand. you're right.
I didn't realize that this server could also be used for non-default APIs.

Reply:
Fix name. https://github.com/autowarefoundation/autoware.universe/pull/704/commits/ef29b63ac5d0ae4e4e3912e0b487d4fc1980d775